doc/reference/bluetooth/hci.txt

```diff
@@ -42,7 +42,8 @@ command is supported.
 	|       | 4    | Set Scan Request Reports                   |
 	|       | 5    | Write Tx Power Level (per Role/Connection) |
 	|       | 6    | Read Tx Power Level (per Role/Connection)  |
-	|       | 7    | Reserved                                   |
+	|       | 7    | Read USB Mode                              |
```

Comment:
Read Supported USB Transport Modes and Set USB Transport Mode

Reply:
Fixed.